modules/claims_api/app/controllers/claims_api/v2/veterans/evidence_waiver_controller.rb

```diff
@@ -13,8 +18,14 @@ def submit
             raise ::Common::Exceptions::ResourceNotFound.new(detail: 'Claim not found')
           end
 
+          unless bgs_claim.dig(:bnft_claim_dto, :filed5103_waiver_ind) == FILE_5103
+            bgs_claim[:bnft_claim_dto][:filed5103_waiver_ind] = FILE_5103
+            response = attempt_submit(bgs_claim)
+          end
+
           EvidenceWaiverBuilderJob.new.perform(target_veteran: target_veteran, response: params[:response])
-          render json: { success: true }
+
+          render json: { success: true, status: 200 } if response == true
```

Comment:
what's the check for here? 
we could also just do `if response` but I don't think it's needed. 
we are raising exceptions throughout the flow so I'm not sure that we need to check the response. Also, if it makes it to this point in the flow, it's similar to v1 and we can just consider it processed, or in process and mark it as success.